projects/packages/sync/src/modules/class-plugins.php

```diff
@@ -287,7 +287,7 @@ public function check_plugin_edit() {
 	 */
 	public function plugin_edit_ajax() {
 		// This validation is based on wp_edit_theme_plugin_file().
-		$args = wp_unslash( $_POST );
+		$args = wp_unslash( $_POST ); // We need all of $_POST since we are validating all args, same as core does in wp_ajax_edit_theme_plugin_file.
```

Comment:
We're not validating all the args though. Just a few of them.

Really, if we want, we could drop this, replace `$args` with `$_POST` everywhere in the function, and add `wp_unslash` on lines 295 and 312 (and maybe 317, although IMO that should be unnecessary their scanner whines about it).

Reply:
Done via https://github.com/Automattic/jetpack/pull/37402/commits/7563c3fa74ad9b98aaf50e41fe19a237a1254f09